src/app/qgsapplayertreeviewmenuprovider.cpp

```diff
@@ -591,6 +592,11 @@ QMenu *QgsAppLayerTreeViewMenuProvider::createContextMenu()
       {
         menu->addSeparator();
         QMenu *menuStyleManager = new QMenu( tr( "Styles" ), menu );
+        QgsMapLayerStyleManager *mgr = layer->styleManager();
+        if ( mgr->styles().count() > 1 )
+        {
+          menuStyleManager->setTitle( tr( "Styles [%1 styles defined]" ).arg( mgr->styles().count() ) );
```

Comment:
What do you think about just "Styles [2]"? The extra "styles defined" text seems extraneous to me.

Reply:
Actually that should probably be "Styles (2)" to match other parts of QGIS

Reply:
Yes, it seems Styles (2) is much better. Updated now. Thanks, Nyall.